contracts/axelar-gateway/src/contract.rs

```diff
@@ -93,13 +94,17 @@ impl AxelarGatewayInterface for AxelarGateway {
             )
     }
 
-    fn is_message_executed(env: Env, source_chain: String, message_id: String) -> bool {
+    /// Return true if a contract call with the given payload BytesN<32> and source caller info has been executed.
```

Comment:
This comment seems a bit off to me. There's no given BytesN<32> payload or source caller for the `is_message_executed` function.

maybe you can reference comments from EVM as well: https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/blob/9847262542dd2279d3a14972f35cf85900657ca9/contracts/gateway/BaseAmplifierGateway.sol#L75